lib/stat_tracker.rb

```diff
@@ -235,4 +235,21 @@ def least_accurate_team(season)
     top_team = @teams.find {|team| team.id == top_team_id }
     top_team.team_name
   end
+
+  def most_tackles(season)
+    season_data = find_season(season)
```

Comment:
I like how you separated out each step like this instead of chaining it together like we did in earlier methods. It's more readable this way, we can plan to refactor other methods similarly.